bin/drebuild

```diff
@@ -1,18 +1,29 @@
 #!/bin/sh
 
 echo "Remove all unused images before we build new ones…"
+echo "\n\033[36m===> docker image prune -f\033[0m\n"
```

Comment:
The colour and line breaks make watching this script run easier to follow and undoubtedly debug. I do think there is a cost of visual noise when working with this file though, feels like I'd have to strip the echo lines to get to grips with the order of things and then add them back in.

Would deduplicating this into a method help with this? Something like so:
```
#!/bin/sh

run_with_pretty_print()
{
  echo "\n\033[36m===> $1\033[0m\n"
  eval $1
}

run_with_pretty_print 'docker image prune -f'
…
```